install/windows/docker-ee.md

```diff
@@ -90,60 +78,6 @@ sconfig
 
 Select option `6) Download and Install Updates`.
 
-## Use a script to install Docker EE
```

Comment:
I don't think we support this for EE, so I removed it. It was telling users to download a file from a url that doesn't seem to exist anymore. https://download.docker.com/components/engine/windows-server/.

Reply:
Hi we were using this in our current setup, because we don't have an internet connection on our servers. Is there an alternative way now?

Reply:
@SbiCA this section was incorrectly removed. https://github.com/docker/docker.github.io/pull/6980 opened to restore it